examples/multiomics_mogonet/model.py

```diff
@@ -0,0 +1,91 @@
+from typing import List, Optional
+
+from torch.nn import CrossEntropyLoss
+from yacs.config import CfgNode
+
+from kale.embed.mogonet import MogonetGCN
+from kale.loaddata.multiomics_gnn_dataset import MogonetDataset
+from kale.pipeline.mogonet_multiomics_trainer import ModalityTrainer
+from kale.predict.decode import LinearClassifier, VCDN
+
+
+class MogonetModel:
```

Comment:
Have you studied the other similar examples? DeepDTA, Polypharmacy, Action.
Yours here is quite different from theirs. Better to follow the same style, which is also neater.

Reply:
Again, this is not yet addressed. Why can't this go into the core API?